backend/pom.xml

```diff
@@ -1,177 +1,173 @@
-            </activation>
-            <properties>
-                <docker.build.skip>true</docker.build.skip>
-            </properties>
-        </profile>
-        <profile>
-            <id>develop</id>
-            <properties>
-                <docker.build.skip>false</docker.build.skip>
-                <docker.build.userid>1001</docker.build.userid>
-            </properties>
-        </profile>
-        <profile>
-            <id>prod</id>
-            <properties>
-                <docker.build.skip>false</docker.build.skip>
-                <docker.build.userid>1001</docker.build.userid>
-            </properties>
-        </profile>        
+		<profile>
+			<id>dev</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<properties>
+				<docker.build.skip>true</docker.build.skip>
+			</properties>
+		</profile>
+		<profile>
+			<id>develop</id>
+			<properties>
+				<docker.build.skip>false</docker.build.skip>
+				<docker.build.userid>1001</docker.build.userid>
+			</properties>
+		</profile>
+		<profile>
+			<id>prod</id>
+			<properties>
+				<docker.build.skip>false</docker.build.skip>
+				<docker.build.userid>1001</docker.build.userid>
+			</properties>
+		</profile>
 
-    </profiles>
+	</profiles>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-data-jpa</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
-        </dependency>
-        <!-- Spring modelmapper for DTO transformation -->
+	<dependencies>
 		<dependency>
-		    <groupId>org.modelmapper</groupId>
-		    <artifactId>modelmapper</artifactId>
-		    <version>2.3.5</version>
-		</dependency>        
-        <dependency>
-            <groupId>com.opencsv</groupId>
-            <artifactId>opencsv</artifactId>
-            <version>4.5</version>
-        </dependency>
-        <dependency>
-            <groupId>org.postgresql</groupId>
-            <artifactId>postgresql</artifactId>
-            <scope>runtime</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>1.18.12</version>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-test</artifactId>
-            <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.junit.vintage</groupId>
-                    <artifactId>junit-vintage-engine</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <!-- Swagger integration for API documentation -->
+			<groupId>org.springframework.boot</groupId>
```

Comment:
Tabs vs Space: We should agree on what we use at least for indention. I suggest 4 spaces instead tabs

Reply:
ok, chenge my IDE from tabs to spaces